### ISSUE TITLE
Individual views hang 

### ISSUE THREAD

**Issue by marius10p** [cortex-lab/phy#848]
This happens relatively often, to one view at a time, and can happen to either one of the views while the other ones are fine. It outputs this message when one view crashes:

QBasicTimer::start: Timers cannot be started from another thread

I am on Win10 and I had to "pip install PyQtWebEngine" to make it work. Closing and re-opening the crashed view works fine.

**Comment by rossant** [comment]
Does it still happen at the same frequency if you use the following plugin (to put in `~/.phy/phy_config.py`)

```python
from phy import IPlugin, connect

c = get_config()

class MyPlugin(IPlugin):
    def attach_to_controller(self, controller):
        @connect
        def on_gui_ready(sender, gui):
            gui._enable_threading = False

c.TemplateGUI.plugins = ['MyPlugin']
```

This disables multithreading for view updates.

**Comment by marius10p** [comment]
It no longer breaks, but it's noticeably slower, about as slow as Phy 1.0.

**Comment by rossant** [comment]
Okay I'll try to investigate further. Note that you can disable auto-update of certain views, menu Views>SomeView>Toggle auto update

**Comment by rossant** [comment]
hopefully fixed with the latest version by 6b08b08e52f473541d9149024a2fcfc5fa699f27
